src/backend/functions/change-requests.ts

```diff
@@ -69,7 +60,7 @@ const changeRequestTransformer = (
   };
   return {
     ...changeRequest,
-    type: convertChangeRequestType(changeRequest.type),
+    type: changeRequest.type,
```

Comment:
This line is erroring saying: 
![image](https://user-images.githubusercontent.com/29521172/150915228-d8ea2768-c95c-4245-a98c-8461cd0d2f86.png)

Reply:
Okay ew. For whatever reason this (the way Prisma implements enums
```ts
const Name = "OPTION_A" | "OPTION_C" | "OPTION_C"
```
is not the same as this (the way we've written our enums)
```ts
enum Name {
  OptionA = 'OPTION_A',
  OptionB = 'OPTION_B',
  OptionC = 'OPTION_C'
}
```

Perhaps we need to keep the converter, but with the renaming and standardization, hopefully it can be closer to a one-liner. @acd124 would probably have a much better idea of what it could look like to just look up the value from one enum in the other enum.

Reply:
https://github.com/microsoft/TypeScript/issues/17690 Basically the type system won't allow us to hand wave it through as it is. I think the cleanest solution might involve changing the ChangeRequestType, maybe involving the prisma type in some way.

Reply:
Okay yeah that's way too much work for such a small thing. Let's try replacing the enum we wrote in `utils` with string literal implementation because I don't want to stir up brining Prisma into the `utils` package

Reply:
Enums give us the flexibility of opaque values, but then we run into issues because Prisma has it implemented as string literal unions

Reply:
> Can we just do a reverse lookup? `Object.values...` this one or the next comment https://github.com/microsoft/TypeScript/issues/17690#issuecomment-351489397

I think the issue is that no matter how we manipulate the value given from prisma short of forcing ts to see it as a member of the enum it will still be a string literal. I think if you want to keep a sort of key value enum shape you might turn it into an object as const and say the enum type is `enumObj[string]` which should create the string union properly, then have values be accessed with `enumObj.name`

Reply:
@acd124 do you want to give it a shot? We can't have Prisma code in the `utils` package and I want to preserve the flexibility that proper enums provide (not having tons of hard-coded strings b/c it being a string literal union). I just wanna see what you can come up with fitting those two requirements.